src/Symfony/Component/Translation/Util/XliffUtils.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Util;
+
+use Symfony\Component\Config\Util\XmlUtils;
+use Symfony\Component\Translation\Exception\InvalidResourceException;
+
+/**
+ * XliffUtils is a bunch of utility methods to Xliff operations.
+ *
+ * This class contains static methods only and is not meant to be instantiated.
+ *
+ * @author Marcos D. Sánchez <marcosdsanchez@gmail.com>
+ */
+class XliffUtils
+{
+    /**
+     * This class should not be instantiated.
+     */
+    private function __construct()
+    {
+    }
+
+    /**
+     * Loads a Xliff file.
+     *
+     * @param string               $file             A Xliff file path
+     * @param string|callable|null $schemaOrCallable An XSD schema file path, a callable, or null to disable validation
+     *
+     * @return \DOMDocument
+     *
+     * @throws \InvalidArgumentException When loading of Xliff file returns error
+     */
+    public static function loadFile($file, $schemaOrCallable = null)
```

Comment:
We need to check that file exist, maybe move code [XliffFileLoader.php#L36-L42](https://github.com/marcosdsanchez/symfony/blob/xliff-utils/src/Symfony/Component/Translation/Loader/XliffFileLoader.php#L36-L42) into this function.